### ISSUE TITLE
v1.5-beta - XOAUTH - aud param is an array

### ISSUE THREAD

**Issue by quenenni** [roundcube/roundcubemail#8123]
Hello,

While trying to connect my Roundcube to my LemonLdap, I found that the value of $body['aud'] is an array, not a string.

In the file program/include/rcmail_oauth.php (line 150), I had to do this change in order to make it works:
```
else if (isset($body['aud']) && $body['aud'] !== $this->options['client_id']) {

by

else if (isset($body['aud']) && $body['aud'][0] !== $this->options['client_id']) {
```

**Comment by alecpl** [comment]
This is fixed in 1.5-rc.

**Comment by thomascube** [comment]
See #8088 and #8096
